workloads/hyper-scale/common.sh

```diff
@@ -51,6 +53,19 @@ setup(){
     rosa verify permissions
 }
 
+pre_flight_checks(){
+    echo "Pre flight checks started"
+    export MULTI_AZ=$(rosa describe cluster -c krvoora-router -o json | jq -r [.multi_az] | jq -r .[])
```

Comment:
you have your cluster name in there still ;-)

Reply:
and on the next line as well

Reply:
Thanks for the catch, updated